e2e-tests/desktop/tests/targets.spec.js

```diff
@@ -89,10 +85,6 @@ test.beforeEach(
       target: targetWithTwoHosts,
       hostSourceIds: [hostSetWithTwoHosts.id],
     });
-    targetWithTwoHosts = await boundaryHttp.addBrokeredCredentials(request, {
```

Comment:
I think we'd still want to test the use case that has a target with two hosts, I think it's fine if one of them just happens to be a brokered credential

Reply:
yeah that makes sense, I modified the tests name for the new tests